### ISSUE TITLE
Is it possible to select a directory to be neovim cwd within vifm?

### ISSUE THREAD

**Issue by satishskamath** [vifm/vifm.vim#86]

**Comment by xaizek** [comment]
No, but it probably won't be hard to add.

**Comment by satishskamath** [comment]
Sorry for the really late reply @xaizek . 
I was wondering how this can be approached? I thought of 2 functions and I am not sure if this can already be achieved in some way other than vifm or default netrw has something built in for this:
* **Function 1**: Setting directory of the currently opened buffer as the `cwd`.
* **Function 2**: Choosing the last opened directory in the file tree as the `cwd`. (https://wiki.vifm.info/index.php/How_to_set_shell_working_directory_after_leaving_Vifm may be using this option `--choose-dir` and passing this on to the editor?)

**Comment by xaizek** [comment]
Yes, need to use result of `--choose-dir`, specifically feed it into `:cd` or `:lcd`.   The catch though is that plugin's code is a mess.  Code-wise:
1. In `s:StartVifm` and `s:HandleRunResults`.
2. Need to check `cd` for `editcmd` parameter and upon detection, pass `--choose-dir` instead of `--choose-files`.
3. `s:HandleRunResults` should do `exec cd <path>` after checking that there is only one path.
